src/components/Editable.js

```diff
@@ -300,10 +299,10 @@ const Editable = ({ disabled, isEditing, thoughtsRanked, contextChain, cursorOff
       // text/plain may contain text that ultimately looks like html (contains <li>) and should be parsed as html
       // pass the untrimmed old value to importText so that the whitespace is not loss when combining the existing value with the pasted value
       const rawDestValue = strip(contentRef.current.innerHTML, { preventTrim: true })
-      importText(thoughtsRankedLive, isHTML(plainText)
+      store.dispatch(importText(thoughtsRankedLive, isHTML(plainText)
```

Comment:
You can also use the `dispatch` variable which we have closure over that is passed to the component from `connect`.